tests/unittests/core/test_apply_func.py

```diff
@@ -28,10 +29,10 @@ class ModelExample:
     label: torch.Tensor
     some_constant: int = dataclasses.field(init=False)
 
-    def __post_init__(self):
+    def __post_init__(self):  # noqa: D105
```

Comment:
(feel free to ignore me) Is this `noqa` necessary still? I see `D105` being removed from `pyproject.toml`
```suggestion
    def __post_init__(self):
```

Reply:
seems yes; otherwise it would be removed by
https://github.com/Lightning-AI/utilities/blob/ffac3e63c467cce0d4f1ddde46764c0ec38febc2/.pre-commit-config.yaml#L51